run-docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run --rm --interactive --tty --device /dev/kvm --user $(id -u) --workdir /recipes --mount "type=bind,source=$(pwd),destination=/recipes" --security-opt label=disable $1 $2
```

Comment:
Please wrap that in multiple lines; and probably just hardcode usage of go-debos/debos as an image name.

 Though as @obbardc mentioned it's a bit of a questionmark who this is for as to get it you'd either need to download it directly or checkout the git repo.. But if you're doing that you may as well just copy&paste the command and tune it. 

I'm not really against it though; for us the only drawback would be the duplication and it potentially going out of sync